docs/components/clients/Logo.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import cn from "classnames";
-import Image from "next/future/image";
+import Image from "next/image";
```

Comment:
This change is actually backwards- and forwards-compatible given our usage.

We'll switch them all over by default when we jump to Next.js 13.

Reply:
Nope, that was because of a local change I made. Reverting this change back out.